infra/prod/s3/azure-blueprint.yaml

```diff
@@ -24,17 +35,19 @@ node_templates:
     properties:
       resource_config:
         environment_variables:
-          ARM_SUBSCRIPTION_ID: { get_secret: azure_subscription_id }
-          ARM_TENANT_ID: { get_secret: azure_tenant_id }
-          ARM_CLIENT_ID: { get_secret: azure_client_id }
-          ARM_CLIENT_SECRET: { get_secret: azure_client_secret }
+          ARM_SUBSCRIPTION_ID: { get_input: [ cloud_credentials, azure_subscription_id ] }
+          ARM_TENANT_ID: { get_input: [ cloud_credentials, azure_tenant_id ] }
+          ARM_CLIENT_ID: { get_input: [ cloud_credentials, azure_client_id ] }
+          ARM_CLIENT_SECRET: { get_input: [ cloud_credentials, azure_client_secret ] }
         variables:
-          blob_name: { get_input: bucket_name }
-          blob_rg: { concat: [{ get_input: resource_prefix}, 'rg'] }
-          blob_location: { get_input: bucket_region }
+          blob_name: { concat: [ { get_attribute: [ prefix, value ] }, bucket ] }
+          blob_rg: { concat: [ { get_attribute: [ prefix, value ] }, rg ] }
+          blob_location: { get_input: [ cloud_credentials, region_name ] }
         source:
           location: https://github.com/jrzeszutek/tf-blob/archive/master.zip
```

Comment:
@jrzeszutek do you think we need to make this part of official repos ?

Reply:
@ahmadiesa-abu I don't have permissions for creating new repos in cloudify-community, so I just moved the zip archives to eaas-example. I think it's not a bad idea to have all of the EaaS code in a single repo